src/assets/osc-bigger.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100"><path fill="none" stroke="#fff" stroke-linecap="round" stroke-linejoin="round" stroke-width="5" d="M92,70L52,30L12,70"/></svg>
```

Comment:
Chevron marks may confuse users who were thinking them as to scroll notes. Modern browsers are using `-` and `+` icons as to scale so we should follow common style by creating new icons.

Reply:
I'll admit I was being lazy and just rotating the ones you already had😂, do you have an easy way to make +&- that match?

Reply:
I made new designs for plus and minus icon at #431.